src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ListAssistHearingMessageHelperTest.java

```diff
@@ -35,19 +36,36 @@ public class ListAssistHearingMessageHelperTest {
     @InjectMocks
     private ListAssistHearingMessageHelper messageHelper;
 
+    private static final String CCD_CASE_ID = "1234";
+
+    @Before
+    public void setUp() {
+        when(hearingMessagingServiceFactory.getMessagingService(LIST_ASSIST))
+            .thenReturn(sessionAwareServiceBusMessagingService);
+    }
+
     @Test
     public void shouldSendExpectedCancellationMessage() {
-        when(hearingMessagingServiceFactory.getMessagingService(LIST_ASSIST)).thenReturn(sessionAwareServiceBusMessagingService);
+        messageHelper.sendListAssistCancelHearingMessage(CCD_CASE_ID, CancellationReason.OTHER);
+        confirmMessageSentWithHearingStateAndCancellationReason(CANCEL_HEARING, CancellationReason.OTHER);
+    }
 
-        messageHelper.sendListAssistCancelHearingMessage("1234", CancellationReason.OTHER);
+    @Test
+    public void shouldSendExpectedCreateMessage() {
+        messageHelper.sendListAssistCreateHearingMessage(CCD_CASE_ID);
+        confirmMessageSentWithHearingStateAndCancellationReason(CREATE_HEARING, null);
+    }
 
+    private void confirmMessageSentWithHearingStateAndCancellationReason(
```

Comment:
same here, this makes the test harder to follow and creates unessisary coupling

Reply:
ok